dwpost/dwpost.php

```diff
@@ -192,7 +193,7 @@ function dwpost_send(App $a, array &$b)
 		Logger::debug('dwpost: data: ' . $xml);
 
 		if ($dw_blog !== 'test') {
-			$x = DI::httpClient()->post($dw_blog, $xml, ['Content-Type' => 'text/xml'])->getBody();
+			$x = DI::httpClient()->post($dw_blog, $xml, HttpClientAccept::DEFAULT, ['Content-Type' => 'text/xml'])->getBody();
```

Comment:
With a `post` we are transmitting the `content-type`, we aren't accepting stuff.